### PR TITLE
Initial react quotes guidelines

### PR DESCRIPTION
**Pull request by lededje**

### CODE REVIEW

**Comment by tommyp** [comment]
LGTM

File: _guidelines/react.md [hunk 1]
```diff
@@ -11,6 +11,27 @@ collection: guidelines
 1. Automatic Table of Contents Here
 {:toc}
 
+## Quotes
+
+Always use double quotes (`"`) for JSX attributes, but single quotes for all other JS. eslint: [`jsx-quotes`](http://eslint.org/docs/rules/jsx-quotes)
```
Comment (kapowaz) [inline _guidelines/react.md]: Footnotes syntax?

File: _guidelines/react.md [hunk 2]
```diff
@@ -11,6 +11,27 @@ collection: guidelines
...
+
+> Why? JSX attributes [can't contain escaped quotes](http://eslint.org/docs/rules/jsx-quotes), so double quotes make contractions like `"don't"` easier to type.
```
Comment (kapowaz) [inline _guidelines/react.md]: Blockquotes

**Comment by martyphee** [comment]
LGTM

**Pull request merged**